File: python_work/220711/accumulator.py
# sum = 0

class Accumulator:
    def __init__(self):
        self.sum = 0

    @staticmethod
    def add(self, i):
        # global sum
        self.sum = self.sum + i

    @staticmethod
    def showResult(self):
        print(f"sum = {self.sum}")

File: java_work/ex0711/src/ex0711/Accumulator.java
package ex0711;

public class Accumulator {

	public static int MAX = 345;
	
	private static int sum = 0;

	public static void add(int i) {
		sum = sum + i;
	}

	public static void showResult() {
		System.out.println("sum = " + sum);
	}

}
